src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/court/commonplatform/CommonPlatformCourtEventListenerIntTest.kt

```diff
@@ -332,37 +332,38 @@ class CommonPlatformCourtEventListenerIntTest : MessagingMultiNodeTestBase() {
 
   @Test
   fun `should process when is youth is null`() {
-    val youthDefendantId = randomDefendantId()
+    val defendantId = randomDefendantId()
     val messageId = publishCommonPlatformMessage(
       commonPlatformHearing(
         listOf(
-          CommonPlatformHearingSetup(defendantId = youthDefendantId, isYouth = null),
+          CommonPlatformHearingSetup(defendantId = defendantId, isYouth = null),
         ),
       ),
 
     )
 
     checkTelemetry(
       MESSAGE_RECEIVED,
-      mapOf("MESSAGE_ID" to messageId, "SOURCE_SYSTEM" to COMMON_PLATFORM.name, "EVENT_TYPE" to COMMON_PLATFORM_HEARING.name),
+      mapOf(
+        "MESSAGE_ID" to messageId,
+        "SOURCE_SYSTEM" to COMMON_PLATFORM.name,
+        "EVENT_TYPE" to COMMON_PLATFORM_HEARING.name,
+        "DEFENDANT_ID" to defendantId,
+      ),
     )
   }
 
   @Test
   fun `should not process when is organisation`() {
-    val youthDefendantId = randomDefendantId()
-    val messageId = publishCommonPlatformMessage(
+    val defendantId = randomDefendantId()
+    publishCommonPlatformMessage(
       commonPlatformHearing(
         listOf(
-          CommonPlatformHearingSetup(defendantId = youthDefendantId, isYouth = null, isPerson = false),
+          CommonPlatformHearingSetup(defendantId = defendantId, isPerson = false),
         ),
       ),
     )
 
-    checkTelemetry(
-      MESSAGE_RECEIVED,
-      mapOf("MESSAGE_ID" to messageId, "SOURCE_SYSTEM" to COMMON_PLATFORM.name, "EVENT_TYPE" to COMMON_PLATFORM_HEARING.name),
-      times = 0,
-    )
+    assertThat(personRepository.findByDefendantId(defendantId)).isNull()
```

Comment:
This will always be true unless the message processing is ⏩ ⏩ 
`awaitAssert` is useful here.